### ISSUE TITLE
Game.Pieces unusable

### ISSUE THREAD

**Issue by Fischchen** [rpkak/sc-client-python#1]
Hey,

I've found that the game.pieces and game.getPiece() throw errors as they are initialized as None.

Furthermore, game.pieces is unusable due to the aforementioned problem.

Is there a workaround, or could you please fix this?

Thanks in advance,
Morris

**Comment by rpkak** [comment]
Do you initialize `Game` manuel?

**Comment by Fischchen** [comment]
No, I'm inside the calculate function. I now seem to be able to loop through game.pieces, but the linter is not happy about it being initialized as None. Is there a way to not initialize it as none?

**Comment by rpkak** [comment]
Ok @Fischchen,

Can you try out, if the linter error exists as well, if `sc-client` uses typing?

Install:
```
pip install git+https://github.com/rpkak/sc-client-python@typing
```

If yes, I will release it soon.

**Comment by rpkak** [comment]
@Fischchen This should be solved in [version 1.2.1](https://github.com/rpkak/sc-client-python/releases/tag/1.2.1).

Please reopen if the linting error exists in [version 1.2.1](https://github.com/rpkak/sc-client-python/releases/tag/1.2.1) as well.
